gocd/gocd_util.go

```diff
@@ -0,0 +1,280 @@
+package gocd
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// Server a representstion of a GoCD server
```

Comment:
Spelling/grammar :)

Reply:
golint has some rules dictating how exported types should be commented. They may not always be english friendly, but I'll keep it in mind when commenting future types/functions